samples/openthread/libraries/prj.conf

```diff
@@ -0,0 +1,77 @@
+#
```

Comment:
Should we modify it and put only all available OT settings (as reported in generated openthread_lib_configuration.txt)?